certora/specs/StakeManager.spec

```diff
@@ -27,30 +27,32 @@ function getAccountBalance(address addr) returns uint256 {
 
 function isMigrationfunction(method f) returns bool {
   return
-          f.selector == sig:migrateFrom(address,bool,StakeManager.Account).selector ||
-          f.selector == sig:migrateTo(bool).selector;
+          f.selector == sig:migrateTo(bool).selector ||
+          f.selector == sig:transferNonPending().selector;
```

Comment:
Needed to remove `migrateFrom` here, because that one shouldn't revert when a migration is happening.